with-gatsby-filesystem-routing/src/pages/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { graphql, Link } from 'gatsby';
+
+function IndexPage({ data: { products } }) {
+  return products.nodes.map((product) => (
+    <Link to={product.gatsbyPath}>{product.name}</Link>
+  ));
+}
+
+export const pageQuery = graphql`
+  query IndexPageQuery {
+    products: allGraphCmsProduct {
+      nodes {
+        name
+        gatsbyPath(filePath: "./{GraphCmsProduct.slug}")
```

Comment:
```suggestion
        gatsbyPath(filePath: "/{GraphCmsProduct.slug}")
```

Must start with a `/` :)

Reply:
Tried this locally, just hadn't committed! Thanks.